cfme/tests/ssui/conftest.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from fixtures.pytest_store import store
+from utils import browser
+
+
+def revert_to_default():
+    store.current_appliance.mode = "default"
+    browser.quit()
+
+
+@pytest.fixture(scope="module", autouse=True)
+def set_ssui_mode(request):
+    store.current_appliance.mode = "ssui"
```

Comment:
As you can see below, I started adding in the menu nodes. The issue is that force_navigate is very tied into the CFME main interface. I wonder if this is just going to have to wait until FW3.0, but there is a part of me that is thinking that we will have multiple Endpoint classes, with their own navigation menus, and their own force_navigates. Perhaps I will try knocking something up on Monday for this.